src/Document/Single.php

```diff
@@ -54,7 +55,7 @@ class Document extends Element
      *
      * @param mixed $id
      *
-     * @return Document
+     * @return $this
```

Comment:
I guess my information is out of date, since that comment was from 2013 and things have moved on a long way. $this it is :+1: